### PR TITLE
[W5.4][W14-A3]Lim Jun Quan

### PR DESCRIPTION
**Pull request by JunQuann**

### CODE REVIEW

File: test/java/seedu/addressbook/commands/SortCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+    private AddressBook initialAddressBook;
+    private AddressBook sortedAddressBook;
+
+    private List<ReadOnlyPerson> sortedDisplayList;
+
+
+    @Before
+    public void setUp() throws Exception {
+        Person johnDoe = new Person(new Name("John Doe"), new Phone("61234567", false),
```
Comment (tshradheya) [inline test/java/seedu/addressbook/commands/SortCommandTest.java]: Could have used the TypicalPersons.java file for already created dummy Person for testing

File: test/java/seedu/addressbook/commands/SortCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+import seedu.addressbook.data.person.*;
+import seedu.addressbook.data.tag.UniqueTagList;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.util.TestUtil;
+
+import java.util.Collections;
+import java.util.List;
+
+
+public class SortCommandTest {
```
Comment (tshradheya) [inline test/java/seedu/addressbook/commands/SortCommandTest.java]: No testing of UI done

File: src/seedu/addressbook/data/AddressBook.java [hunk 3]
```diff
@@ -83,6 +83,10 @@ public void addPerson(Person toAdd) throws DuplicatePersonException {
         syncTagsWithMasterList(toAdd);
     }
 
+    public void sortPerson() {
+        allPersons.sort();
+    }
+
```
Comment (alvinlyj) [inline src/seedu/addressbook/data/AddressBook.java]: Perhaps you can give some comments above the method to give a summary or some insights to the method you have implemented

File: test/java/seedu/addressbook/commands/SortCommandTest.java [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        assertEquals(expectedAddressBook.getAllPersons(), actualAddressBook.getAllPersons());
+    }
+
+    private SortCommand createSortCommand() {
+        SortCommand command = new SortCommand();
+        command.setData(initialAddressBook, Collections.emptyList());
+
+        return command;
+    }
+}
```
Comment (alvinlyj) [inline test/java/seedu/addressbook/commands/SortCommandTest.java]: Perhaps you can add some comments above the methods to give a brief summary on the function/purpose of the methods

**Comment by alvinlyj** [comment]
Work is good and clean. You have also tackled all that is required for this LO. 

Area for improvement: Test cases can be more extensive (i.e. more kinds of test cases that test corner cases of the code implemented. (However not expected now as formation of test cases will only be covered in later LOs, just take note for your own improvement in the future.)

Do take note of comments given and close the PR once you have reviewed the comments.

File: src/seedu/addressbook/parser/Parser.java [hunk 5]
```diff
@@ -11,17 +11,7 @@
...
-import seedu.addressbook.commands.DeleteCommand;
-import seedu.addressbook.commands.ExitCommand;
-import seedu.addressbook.commands.FindCommand;
-import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
-import seedu.addressbook.commands.ListCommand;
-import seedu.addressbook.commands.ViewAllCommand;
-import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.*;
 import seedu.addressbook.data.exception.IllegalValueException;
```
Comment (alvinlyj) [inline src/seedu/addressbook/parser/Parser.java]: Don't import using * import the classes explicitly, only what is necessary
